pkg/game/game.go

```diff
@@ -161,6 +167,20 @@ func (game *Game) CurrentPlayer() elements.Player {
 	return game.players[game.currentPlayer]
 }
 
+func (game *Game) GetPlayerByID(playerID elements.ID) elements.Player {
```

Comment:
Add a comment that this is only meant to be used by tests - usage panics could be problematic otherwise.

Reply:
commented